test/utils.js

```diff
@@ -0,0 +1,265 @@
+'use strict';
```

Comment:
Note to reviewers:

The purpose of this new test file is to provide full test-coverage of the `./cli/utils.js` file. I know there's still a lot of untested code. This project didn't have any tests at all and I've slowly started adding tests to improve the situation. Previously with the `./test/config.js` file and today with this file.

Reply:
Thanks a lot for adding these tests!